bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,154 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+## Supported Things
+
+* EnergyManager  
+  Location  
+  Forecast  
+  GridFlow  
+  ProfileApp  
+  ScheduleApp  
+  SimpleSwitcher (seems deprecated)  
+  SmartEnergyManagement (seems deprecated)
+
+* BatteryConverter
+  
+* MyReserve  
+  MyReserveInverter  
+  MyReservePowerMeter
+  
+* EVStation (eg. Keba)
+* S0Counter
+* SunSpecInverter (eg. Fronius)  
+
+
+## Discovery
+
+You have to enter the hostname or ip-address of the energymanager itself.
+The attached devices and supported channels are discovered automatically.
+
+## Binding Configuration
+
+The binding has no configuration settings.
+
+## Thing Configuration
+
+Only the EnergyManager thing requires configuration.
+You have to give it the hostname or ip-address.
+
+## Channels
+
+### All devices
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| stateDevice | String | value reflecting the state of the communication from the energy manager to the device. *ON* or *OFFLINE* |
```

Comment:
Isn't this identical to the Thing status?

Reply:
This is the internal communication state from the energymanager to the connected devices. It is not the same, as the ThingStatus is still ONLINE even so the energymanager had a communication timeout.

Reply:
I understood that `energymanager` is the Bridge and all other Things are connected to the `energymanager`. If that is correct, the Things status of e.g. `pvplant` should be OFFLINE when then connection between `energymanager` and `pvplant` is down.

> the ThingStatus is still ONLINE even so the energymanager had a communication timeout.

Do you mean a connection timeout between `energymanager` and openHAB? The Thing statuses of the child Things are set to OFFLINE by the framework automatically when the Bridge is OFFLINE.

Reply:
Sounds reasonable. Can the Channel be removed, then?